sim/resources/stompymicro/joints.py

```diff
@@ -0,0 +1,210 @@
+        Left = "Hip_Pitch_Left"
+        Right = "Hip_Pitch_Right"
+    
+    Lift_Joint_Limit_2 = "Hip_Lift_Joint_Limit_2"
+
+class RightLeg(Node):
+    hip_pitch = "Hip_Pitch_Right"
+    hip_lift = "Hip_Lift_Joint_Limit_2"
+    hip_roll = "Thigh_Rotate_2"
+    knee_rotate = "Knee_Rotate_2"
+    foot_rotate = "Foot_rotate_2"
+
+class LeftLeg(Node):
+    hip_pitch = "Hip_Pitch_Left"
+    hip_lift = "Hip_Lift_Joint_Limit"
+    hip_roll = "Thigh_Rotate"
+    knee_rotate = "Knee_Rotate"
+    foot_rotate = "Foot_rotate"
+
+class Legs(Node):
+    left = LeftLeg()
+    right = RightLeg()
+
+class Robot(Node):
+    height = 0.21
+    rotation = [0.0, 0.0, 0, 1]
+
+    # TODO
+    # collision_links = [
+    #     "lower_half_assembly_1_left_leg_1_foot_pad_1_simple",
+    #     "lower_half_assembly_1_right_leg_1_foot_pad_1_simple",
+    # ]
+
+    legs = Legs()
+
+    @classmethod
+    def default_standing(cls) -> Dict[str, float]:
+        return {
+            cls.legs.left.hip_pitch: 0,
+            cls.legs.right.hip_pitch: 0,
+            cls.legs.left.hip_lift: 0,
+            cls.legs.right.hip_lift: 0,
+            cls.legs.left.hip_roll: 0,
+            cls.legs.right.hip_roll: 0,
+            cls.legs.left.knee_rotate: 0,
+            cls.legs.right.knee_rotate: 0,
+            cls.legs.left.foot_rotate: 0,
+            cls.legs.right.foot_rotate: 0,
+        }
+
+    @classmethod
+    def default_limits(cls) -> Dict[str, Dict[str, float]]:
+        return {
+            # left leg
+            cls.legs.left.hip_pitch: {
+                "lower": -1.5707963,
+                "upper": 1.5707963,
+            },
+            cls.legs.left.hip_lift: {
+                "lower": -0.087266463,
+                "upper": 1.5707963,
+            },
+            cls.legs.left.hip_roll: {
+                "lower": -0.52359878,
+                "upper": 0.52359878,
+            },
+            cls.legs.left.knee_rotate: {
+                "lower": -1.5707963,
+                "upper": 1.5707963,
+            },
+            cls.legs.left.foot_rotate: {
+                "lower": -1.5707963,
+                "upper": 1.5707963,
+            },
+            # right leg
+            cls.legs.right.hip_pitch: {
+                "lower": -1.5707963,
+                "upper": 1.5707963,
+            },
+            cls.legs.right.hip_lift: {
+                "lower": -1.5707963,
+                "upper": 0.087266463,
+            },
+            cls.legs.right.hip_roll: {
+                "lower": -0.52359878,
+                "upper": 0.52359878,
+            },
+            cls.legs.right.knee_rotate: {
+                "lower": -1.5707963,
+                "upper": 1.5707963,
+            },
+            cls.legs.right.foot_rotate: {
+                "lower": -1.5707963,
+                "upper": 1.5707963,
+            },
+        }
+
+    # p_gains
+    @classmethod
+    def stiffness(cls) -> Dict[str, float]:
+        return {
+            "hip pitch": 250,
```

Comment:
Update the names to:
```suggestion
            "Hip_Pitch": 250,
```
if you carefully look at you initialization you don't use any of your joints.